apps/common/utils/index.tsx

```diff
@@ -70,11 +70,11 @@ export function getAmountWithSlippage(from: string, to: string, value: BigNumber
 	return formatToNormalizedValue(value || ethers.constants.Zero, 18);
 }
 
-export function handleInputChange(
-	e: React.ChangeEvent<HTMLInputElement>,
+export function toNormalizedBN(
+	value: string,
 	decimals: number
 ): TNormalizedBN {
-	let		amount = e.target.value.replace(/,/g, '.').replace(/[^0-9.]/g, '');
+	let		amount = value.replace(/,/g, '.').replace(/[^0-9.]/g, '');
```

Comment:
The change is similar to this, it's using the `value` rather than the event, it's already merged so we can just reuse

https://github.com/yearn/web-lib/pull/156/files#diff-34b253f6b41c97d2870b0de3bbfc44697ea3da921a051f46ba4a4ef235adf7af